forge/files/src/index.js

```diff
@@ -43,13 +26,22 @@ app.on('window-all-closed', () => {
   }
 });
 
-app.on('ready', () => {
+app.on('ready', async () => {
   if (isDev) {
-    require('devtron').install();
-    installExtension(EMBER_INSPECTOR)
-      .catch((err) => console.log('An error occurred: ', err));
+    try {
```

Comment:
I don't feel too strongly about this, but it seems kinda lousy to prevent the whole app from launching if one of these development tools fails to load for whatever reason. Also, if something does fail, who knows what the error will look like and if it will be easy for users to figure out what's going on...

So, that was my thinking -- control the messaging and make it non-fatal.

Reply:
Sure, we can keep it 👍